portfolio/src/main/java/com/google/sps/servlets/date.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Date;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that responds with the current date. */
+@WebServlet("/date")
+public class date extends HttpServlet {
```

Comment:
Java classes should be capitalized (Date instead of date).

Reply:
Got it